addons/account/models/account_move.py

```diff
@@ -1198,11 +1198,20 @@ def unlink(self):
             self.env['account.move'].browse(list(move_ids))._post_validate()
         return result
 
+    def _get_updated_fields(self, vals):
+        '''Return True if a field are modified'''
+        for key in [k for k in vals.keys() if k in ('account_id', 'journal_id', 'date', 'move_id', 'debit', 'credit')]:
+            for record in self:
+                if key in ('account_id', 'journal_id', 'move_id') and record[key].id != int(vals[key])\
```

Comment:
not checking all the same fields from line 1203?

Reply:
```python
if key in ('account_id', 'journal_id', 'move_id') and record[key].id != int(vals[key]) or record[key] != vals[key]:
    return True
```
it is the same as:
```python
if key in ('account_id', 'journal_id', 'move_id'):
    if record[key].id != int(vals[key]):
           return True
else:
    if record[key] != vals[key]:
         return True
```